wfsim/strax_interface.py

```diff
@@ -266,8 +248,8 @@ def set_timing(self,):
     
     def set_configuration(self,):
         """Set chunking configuration and feeds instructions to wfsim"""
-        max_events_per_chunk=500#is this hard coding really nessecairy...
-        chunk_size = np.ceil(max_events_per_chunk/self.context.config['event_rate'])
+        max_events_per_chunk=20 #is this hard coding really nessecairy...
```

Comment:
@petergaemers Can you help me understand if this can become problematic for very high rates?